perlmod/Fink/Services.pm

```diff
@@ -594,6 +599,32 @@ EOSCRIPT
 		@wrap = map "$_=$ENV{$_}", sort keys %ENV;
 		push @wrap, "__CFPREFERENCES_AVOID_DAEMON=1";
 		unshift @wrap, 'env' if @wrap;
+		my $runtime_request = Fink::Config::get_option("build_in_sandbox");
```

Comment:
This block of code to implement precedence of various controls is poor. Fink::Config::get_option() is documented to take an optional default value, so you don't need to do a ton of if/then to check whether known values are assigned before falling through to default values.